lib/spack/spack/compilers/__init__.py

```diff
@@ -40,6 +40,7 @@
 
 _imported_compilers_module = 'spack.compilers'
 _path_instance_vars = ['cc', 'cxx', 'f77', 'fc']
+_flags_instance_vars = ['cflags', 'cxxflags', 'fflags']
```

Comment:
The fflags are required by the xlf* compiler to work properly.  I see now the following definitions in spec.py:
_valid_compiler_flags = [
    'cflags', 'cxxflags', 'fflags', 'ldflags', 'ldlibs', 'cppflags']

I will add cppflags.  Do I do a new commit on my repo and a new pull request?!?